### DISCUSSION TITLE
Brand new eval install fails at the very end. Says had a problem and not loading the website. /root/errors.log has no errors.

### DISCUSSION THREAD

**Discussion by charlespctech** [Security-Onion-Solutions/securityonion#5345] [Unsupported Versions] [answered]
My Installation fails and I am unable to load the site at all. 

Version 2.3.70

Installed from security onion ISO

EVAL mode

ESXI Host

DHCP reservation set on router 192.168.0.79/24
Gateway 192.168.0.1
DNS is ISP

[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/7080591/sosetup.log)

[admin@securityonion ~]$ sudo so-status
[sudo] password for admin:

Checking Docker status

    Docker -------------------------------------------------------------------------------------------------- [ OK ]

Checking container statuses

    so-aptcacherng ------------------------------------------------------------------------------------------ [ OK ]
    so-cortex ----------------------------------------------------------------------------------------------- [ OK ]
    so-curator ---------------------------------------------------------------------------------------------- [ OK ]
    so-dockerregistry --------------------------------------------------------------------------------------- [ OK ]
    so-elastalert ------------------------------------------------------------------------------------------- [ OK ]
    so-elasticsearch ---------------------------------------------------------------------------------------- [ OK ]
    so-filebeat --------------------------------------------------------------------------------------------- [ OK ]
    so-fleet ------------------------------------------------------------------------------------------------ [ OK ]
    so-grafana ---------------------------------------------------------------------------------------------- [ OK ]
    so-idstools --------------------------------------------------------------------------------------------- [ OK ]
    so-influxdb --------------------------------------------------------------------------------------------- [ OK ]
    so-kibana ----------------------------------------------------------------------------------------------- [ OK ]
    so-kratos ----------------------------------------------------------------------------------------------- [ OK ]
    so-mysql ------------------------------------------------------------------------------------------------ [ OK ]
    so-nginx ------------------------------------------------------------------------------------------------ [ OK ]
    so-playbook --------------------------------------------------------------------------------------------- [ OK ]
    so-redis ------------------------------------------------------------------------------------------------ [ OK ]
    so-sensoroni -------------------------------------------------------------------------------------------- [ OK ]
    so-soc -------------------------------------------------------------------------------------------------- [ OK ]
    so-soctopus --------------------------------------------------------------------------------------------- [ OK ]
    so-steno ------------------------------------------------------------------------------------------------ [ OK ]
    so-strelka-backend -------------------------------------------------------------------------------------- [ OK ]
    so-strelka-coordinator ---------------------------------------------------------------------------------- [ OK ]
    so-strelka-filestream ----------------------------------------------------------------------------------- [ OK ]
    so-strelka-frontend ------------------------------------------------------------------------------------- [ OK ]
    so-strelka-gatekeeper ----------------------------------------------------------------------------------- [ OK ]
    so-strelka-manager -------------------------------------------------------------------------------------- [ OK ]
    so-suricata --------------------------------------------------------------------------------------------- [ OK ]
    so-telegraf --------------------------------------------------------------------------------------------- [ OK ]
    so-thehive ---------------------------------------------------------------------------------------------- [ OK ]
    so-thehive-es ------------------------------------------------------------------------------------------- [ OK ]
    so-wazuh ------------------------------------------------------------------------------------------------ [ OK ]
    so-zeek ------------------------------------------------------------------------------------------------- [ OK ]
    
    
[salt-callstate.highstate.log](https://github.com/Security-Onion-Solutions/securityonion/files/7080622/salt-callstate.highstate.log)

**Comment by charlespctech** [comment, marked as answer]
reinstalling with newest iso with wazuh hotfix resolved my issue

**Reply by sudo-jake** [reply]
Same issue. I'll give this a shot. 

Thank You.

**Reply by dougburks** [reply]
Yep, from https://blog.securityonion.net/2021/08/security-onion-2370-wazuh-hotfix-now.html:

> After we released Security Onion 2.3.70, there was a change on the upstream Wazuh repo that caused our installer to attempt to update Wazuh which resulted in an error. We've added some logic to prevent this from happening.
